### ISSUE TITLE
"No Exit" sign for Sherman St. 

### ISSUE THREAD

**Issue by cpenny86** [Ambridge/Ambridge#68]
Since the hotel went in, the exit from Sherman St. around the Trinity parking lot to 11th is inaccessible. I can't tell you how many cars I've seen drive down expecting to still be able to use it, only to have to execute a 3(+)-point turn or back out. A sign would help!

**Comment by jmkauer** [comment]
Good Morning,

The Ambridge Public Works Dept. did install a NO OUTLET Sign on Sherman
Street the month that the new hotel opened.  Its located near Merchant
Street as soon as you enter Sherman.

Joe Kauer



Joseph Kauer
*Borough Manager*
*BOROUGH OF AMBRIDGE*
600 Eleventh Street
Ambridge, PA  15003
P:  724-266-4070
F:  724-266-9191
www.AmbridgeBoro.org

**Comment by whit537** [comment]
Can you confirm, @cpenny86? If so we're good to close this ticket.

**Comment by cpenny86** [comment]
Yes, I did see it there today. It is very difficult to see if you're driving North on Merchant, however, as there is a tree in the way -- and I wonder if that's part of why people still head down Sherman and have to turn back. Lowering it about a foot (similar to the lower height of the same sign at Merchant & 12th) would likely help a great deal.
